app/mailers/invite_people_mailer.rb

```diff
@@ -11,14 +11,15 @@ def to_start_group(invitation:,
                      subject_params: {group_name: @invitation.invitable_name}
   end
 
-  def to_join_group(invitation: , locale: )
+  def to_join_group(invitation:, locale:, subject:)
     @invitation = invitation
+    invitation.increment_send_count!
```

Comment:
Should we wait to increment the send count until after the `send_single_mail` call?